### ISSUE TITLE
GUI

### ISSUE THREAD

**Issue by sinancang** [sinancang/pomodoro#5]
A graphic user interface that ties in all the functionalities for the program.

**Comment by Minecraftian14** [comment]
Do you need a Swing app for this program? Might have to change the program a bit. Give a sample drawing of the layout you expect and I will make it.

**Comment by sinancang** [comment]
Definitely! Sorry for the late reply, but it definitely is very encouraging seeing some interest in the app. I'm currently working on the layout and will come back with it as soon as it's done.

**Comment by Minecraftian14** [comment]
Opps, I am sorry, I just deleted the message giving up hope, just after which your message popped up.
Yeah sure, i would like to help in your project.

**Comment by Minecraftian14** [comment]
So you gone for another month? 😁

**Comment by sinancang** [comment]
Ahahahaha nice one.

I haven't really thought of a style for the layout, but I have something in mind. When the app opens, the user should see the timer set to 25:00 right in the middle of the screen, with a simple unicolor background. On the right-down corner there should be two buttons, a preferences button (a gear icon perhaps?) and a statistics button. When the user clicks the gear, the app will redirect to the page where the user can set how long the timer should go for, which project the timer should count towards and other necessary preferences. For the other one, the user will be redirected to the page where he can see relevant stats like mean time spent working, pie chart of time project distribution etc etc.

**Comment by Minecraftian14** [comment]
Wow, that's a lot of planning done... Okay, I'll try to do it but there are two things:
-> Please don't vanish again, you have to stay in contact (If not comfortable, I am also on Discord).
-> I'll make the GUI in JavaFX, which will require the project to be upgraded as a Gradle project.

**Comment by Minecraftian14** [comment]
Btw, can you use IntelliJ IDEA? Not necessary but very useful.

**Comment by Minecraftian14** [comment]
And which version of java are you using?

**Comment by Minecraftian14** [comment]
...Give me permissions to commit changes.

**Comment by sinancang** [comment]
> Btw, can you use IntelliJ IDEA? Not necessary but very useful.

yes that's what i'm using

> And which version of java are you using?

i'm using 12.02.2

> ...Give me permissions to commit changes.

sure thing.

**Comment by Minecraftian14** [comment]
Okay, so I have made a lot of changes. Go through them, and if you want to, merge them.
Ask me anything you want to ask. Plus, inform me, so I can start working on S2.

**Comment by sinancang** [comment]
> Okay, so I have made a lot of changes. Go through them, and if you want to, merge them.
> Ask me anything you want to ask. Plus, inform me, so I can start working on S2.

hey, rookie question, where can i see the changed code?

**Comment by Minecraftian14** [comment]
<img width="1036" alt="Temp" src="https://user-images.githubusercontent.com/52451860/86429059-fd4a0d00-bd0b-11ea-83ab-e2027f8e0d24.png">

**Comment by Minecraftian14** [comment]
...
Did it grew too hard? Sorry for that. But giving it a basic infra was necessary to proceed on the project.
